### DISCUSSION TITLE
Get existing Editor instance (from another context)

### DISCUSSION THREAD

**Discussion by TheDigitalOrchard** [GrapesJS/grapesjs#5878] [Q&A]
I have a second JavaScript file that I need to work with my editor instance. I've searched extensively, but it seems that the only way to get the Editor instance is via `init()`, but I want to get the instance after it has already been initialized in a different script.

Is there an obvious method that I've missed?

I do have a custom solution in mind, but wanted to ask about an official method before implementing something custom.

Thanks for any advice.

**Comment by artf** [comment]
I'd say it's up to you to decide where to store your editor instance. The proper way would be a global state manager which could be accessed around your application.
